app/data/repositories/CardsRepository.py

```diff
@@ -20,8 +22,11 @@ def create(deck_id, word, translation) -> Optional[Card]:
     card_id = c.lastrowid
 
     if card_id:
+        logger.debug(
+            f"Card with id {card_id} has been saved into database successfully")
         return Card(card_id, deck_id, word, translation)
 
+    logger.error("Saving card into database failed")
```

Comment:
there is no exception? are you able to log error from db?

Reply:
I think I can do this, implement exception handling

Reply:
DO IT or add todo

Reply:
added TODO, done